src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs

```diff
@@ -38,11 +45,18 @@ internal static bool Filter(string category, LogLevel actualLevel, LogLevel minL
 
         private static bool IsFiltered(string category)
         {
-            ImmutableArray<string> systemLogCategoryPrefixes = FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableHostLogs)
-                                                                ? ScriptConstants.SystemLogCategoryPrefixes
-                                                                : ScriptConstants.RestrictedSystemLogCategoryPrefixes;
+            return _filteredCategoryCache.GetOrAdd(category, c => _cachedPrefixes.Any(p => category.StartsWith(p)));
```

Comment:
I am assuming `c` is the same as `category`? If so, this is a perf improvement to avoid a lambda closure:

```suggestion
            return _filteredCategoryCache.GetOrAdd(category, c => _cachedPrefixes.Any(p => c.StartsWith(p)));
```

Reply:
That's correct, thanks for the tip!